server/sonar-web/src/main/js/apps/projectActivity/components/GraphsHistory.js

```diff
@@ -102,7 +106,7 @@ export default class GraphsHistory extends React.PureComponent {
     return [];
   };
 
-  updateTooltip = (selectedDate: ?Date) => this.setState({ selectedDate });
+  updateTooltip = (selectedDate /*: ?Date */) => this.setState({ selectedDate });
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)